frontend/package.json

```diff
@@ -10,10 +10,10 @@
   "workspaces": {
     "packages": [
       "packages/*"
-    ],
-    "nohoist": [
-      "**"
     ]
   },
-  "dependencies": {}
+  "dependencies": {},
+  "resolutions": {
+    "babel-loader": "8.1.0"
+  }
```

Comment:
this `babel-loader` resolution was added to resolve the following problem: https://github.com/storybookjs/storybook/issues/5183